upstream/upaws/aws.go

```diff
@@ -0,0 +1,98 @@
+package upaws
+
+import (
+	"os"
+	"text/template"
+)
+
+const (
+	AccessEnvVar          = "MM_APPS_AWS_ACCESS_KEY"           // nolint:gosec
```

Comment:
@levb will this be used for APPs invoke?

Reply:
Yes